### PR TITLE
Add launcher template tags

### PR DESCRIPTION
**Pull request by mcmikashi**
Add the launcher template tags

### CODE REVIEW

File: transport_nantes/topicblog/templatetags/test_templatags.py [hunk 1]
```diff
@@ -94,3 +96,62 @@ def test_email_cta_button(self):
...
+            user=self.admin,
+            article_slug=self.item.slug,
+            template_name="topicblog/content_launcher.html",
+            headline="Headline")
+        self.launcher.save()
+
+    def test_launcher(self):
+        url = reverse_lazy("topic_blog:view_item_by_slug",
+                           args=[self.launcher.article_slug])
+        exepted_template =\
```
Comment (Shriukan33) [inline transport_nantes/topicblog/templatetags/test_templatags.py:133]: excepted

**Review by Shriukan33 — APPROVED** [review]
Except for the typo, LGTM

**Comment by JeffAbrahamson** [comment]
Mostly looks good, but two issues jump out for me:

1) The template doesn't seem to resemble what we're currently doing (cf `asso_tn/templates/asso_tn/mpanel/projects.html`).

2) The test is checking for the precise HTML in the template.  This is overly specific.  If we change something about the presentation, the test will fail.  But we want to check that the thing is doing _functionally_ what we want, not that it's bit-for-bit the same forever.  So pick out things like the title and the image and such and make sure that they are present in elements that are tagged as you expect them to be tagged.  Just don't do a full string match on the whole block of HTML.

In general, you want to test _functionality_ and not _implementation_.

**Comment by JeffAbrahamson** [comment]
LGTM.

**Comment by mcmikashi** [comment]
- update the test

**Pull request merged**